src/Compilers/CSharp/Portable/SymbolDisplay/ObjectDisplay.cs

```diff
@@ -133,115 +133,115 @@ internal static string FormatLiteral(bool value)
             return value ? "true" : "false";
         }
 
-        internal static string FormatString(string str, char quote, bool escapeNonPrintable)
+        private static void FormatStringChar(
+            ref PooledStringBuilder pooledBuilder,
+            string str,
```

Comment:
Will leave as is - now that the method is named FormatStringChar, the one string is more obviously the full string.